### PR TITLE
Added Sunfuse 6HP

### PR DESCRIPTION
**Pull request by crackerizer**
Added Sun fuse 6HP, https://www.es.co.th/Schemetic/PDF/6H_6HP.PDF

------------

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [x] If there are matching symbol or 3D model PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required

### CODE REVIEW

**Comment by poeschlr** [comment]
There are a few things that need to be corrected. Please add a 3d model reference.
The courtyard clearance should be 0.25 all around
Some measurements of the fab outline are not quite right.
Move the Silk outline outside the fab outline.

screenshot from 2018-01-18 01-01-48

**Comment by crackerizer** [comment]
There is no 3d model for it. Can I use a dummy one or I have to create one for it?

**Comment by jkriege2** [comment]
you don't need to create a model, there just needs to be the correctly set link in the footprint file, so a 3D model is automatically associated once somebody creates it (or has it available locally).

**Comment by crackerizer** [comment]
I think I got everything fixed. Please check.

File: Fuse.pretty/Fuse_SunFuse-6HP.kicad_mod [hunk 1]
```diff
@@ -0,0 +1,48 @@
+(module Fuse:Fuse_SunFuse-6HP (layer F.Cu) (tedit 5A64049B)
```
Comment (poeschlr) [inline Fuse.pretty/Fuse_SunFuse-6HP.kicad_mod]: Did you export the footprint from the footprint browser and copy the footprint by hand into the .pretty repo?

Please remove "Fuse:"

**Comment by crackerizer** [comment]
Yes, I did copy from the archive lib and forgot to change the name.
It is fixed.

**Comment by evanshultz** [comment]
@jkriege2 @poeschlr 
Can you check this again and merge if it's ready?

**Comment by poeschlr** [comment]
Looks good.

**Comment by crackerizer** [comment]
Thanks guys for making it happen.

**Pull request merged**